File: bai_kiem_tra_02/lib/models/court.dart
class Court {
  final int id;
  final String name;
  final double pricePerHour;
  final bool isActive;

  Court({
    required this.id,
    required this.name,
    required this.pricePerHour,
    required this.isActive,
  });

  factory Court.fromJson(Map<String, dynamic> json) {
    return Court(
      id: json['id'],
      name: json['name'],
      pricePerHour: (json['pricePerHour'] as num).toDouble(),
      isActive: json['isActive'],
    );
  }
}

File: Pcm.Backend/Pcm.Api/Models/Entities/Court.cs
namespace Pcm.Api.Models.Entities
{
    public class Court
    {
        public int Id { get; set; }

        public string Name { get; set; } = null!;

        public decimal PricePerHour { get; set; }

        public bool IsActive { get; set; } = true;
    }
}
